doc/source/release-notes/Percona-Server-5.7.20-19.rst

```diff
@@ -0,0 +1,63 @@
+.. rn:: 5.7.20-19
+
+========================
+Percona Server 5.7.20-19
+========================
+
+Percona is glad to announce the release of Percona Server 5.7.20-19 
+on January 3, 2018. Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.7/Percona-Server-5.7.20-19/>`_
+and from the :doc:`Percona Software Repositories </installation>`.
+
+This release is based on `MySQL 5.7.20
+<http://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-20.html>`_
+and includes all the bug fixes in it.
+Percona Server 5.7.20-19 is now the current GA (Generally Available) release 
+in the 5.7 series. All software developed by Percona is open-source and free.
+Details of this release can be found in the `5.7.20-19 milestone on Launchpad
+<https://launchpad.net/percona-server/+milestone/5.7.20-19>`_.
+
+New Features
+============
+
+* Now MyRocks Storage Engine has *General Availablity* status.
+
+* Binary log encryption has been implemented and can be now switched on using
+  the :variable:`encrypt_binlog` variable.
+
+* :variable:`print_lock_wait_timeout_info` variable, introduced in previous 
+  release, but undocumented, allows to log information about all lock wait 
+  timeout errors.
+
+Bugs Fixed
+==========
+
+* Intermediary slave with Blackhole storage engine couldn't record updates 
+  from master to its own binary log in case of activated informational row-based 
+  logging. Bug fixed :bug:`1722789`
+
+* Help command provided a link to an older version of the |Percona Server| manual.
+  Bug fixed :bug:`1708073`
+
+* A regression in the ``mysqld_safe`` script prevented to stop MySQL service correctly.
```

Comment:
I cannot tell from the bug, what does "prevented to stop correctly" mean? Did it not stop? Was there simply an extra error printed?

Should it be "prevented stopping", instead of "prevented to stop?"